functions-and-scope.js

```diff
@@ -16,6 +17,21 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+// * Door de array met cijfers kunnen itereren: for loop gebruiken, conditie: het cijfer moet 8 of hoger dan 8 zijn >=8
+// * Als je arraynaam.length property gebruikt dan werkt het ook bij meerder entries
+// * Er moet iets terug worden gegeven in variabel
+
+// const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
+
+let counter = 0;
+
+for (let i = 0; i < grades.length; i++) {
+    if (grades[i] >= 8) {
+        counter = counter + 1;
```

Comment:
Dit zou je iets kunnen versimpelen
```suggestion
        counter++;
```